pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -114019,7 +114019,7 @@ self: {
   "gi-poppler" = callPackage
     ({ mkDerivation, base, bytestring, Cabal, containers, gi-cairo
      , gi-gio, gi-glib, gi-gobject, haskell-gi, haskell-gi-base
-     , haskell-gi-overloading, poppler_gi, text, transformers
+     , haskell-gi-overloading, poppler, text, transformers
```

Comment:
@NixOS/haskell does this need some update in the tooling?

Reply:
https://github.com/NixOS/cabal2nix/blob/4541e7090bd30063db086e6e9c3dc1dbcf8526a8/cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/Name.hs#L142